build.sbt

```diff
@@ -16,9 +16,9 @@ lazy val backEnd = project.in(file("."))
 //    )
 //  )
 
-//might need to put common types (like memberId...
-lazy val commonTypes = project.in(file("commonTypes"))
+// This is for types defined at 'domain' scope and having cross-service applicability only.
+lazy val commonTypes = project.in(file("common-types"))
   .configure(C.akkaPersistentEntity("improving-app-common-types"))
```

Comment:
So I think this one needs to use C.protobufsLib. It is not creating an entity, just protobuf types shared between subprojects.

Reply:
Is this module specifically just protobufs? Or are there also scala classes included in here?

Reply:
I would say it is mostly protobufs. With ScalaPB there might be some TypeMaps in Scala that are used from the generated code. But there shouldn't be any Scala not related to the protobuf type definitions.

Reply:
applied changes already 👍